action.yml

```diff
@@ -84,14 +94,14 @@ runs:
           --fail-at-end \
         ${{ inputs.phases }}
       env:
-        # AWS SDK v1
-        AWS_ACCESS_KEY: ${{ inputs.maven-aws-access-key }}
-        AWS_SECRET_KEY: ${{ inputs.maven-aws-secret-key }}
-        # AWS SDK v2
-        AWS_ACCESS_KEY_ID: ${{ inputs.maven-aws-access-key }}
-        AWS_SECRET_ACCESS_KEY: ${{ inputs.maven-aws-secret-key }}
-        # internal property used in assume role AWS credential provider
-        AWS_ASSUME_ROLE_ARN: ${{ inputs.maven-aws-role }}
+        MAVEN_AWS_PROFILE: ${{ inputs.maven-aws-profile || env.MAVEN_AWS_PROFILE }}
+        AWS_PROFILE: ${{ inputs.maven-aws-profile || env.MAVEN_AWS_PROFILE }} # for now the same as MAVEN_AWS_PROFILE, otherwise we need to assume new role from another OIDC profile
```

Comment:
MAVEN_AWS_PROFILE is intended to be used only with wagon directly. AWS_PROFILE is intended to be default for SDK for other things during build like: maven plugins. Although the most probably it will be always the same profile.